spec/21-http_response_header_format.md

```diff
@@ -53,7 +53,7 @@ trace-flags      = 2HEXDIGLC   ; 8 bit flags. See below for details
 
 The format and requirements for this are the same as those of the trace-id field in the `traceparent` request header.
 
-For details, see the trace-id section under [traceparent Header Field Values](#traceparent-Header-Field-Values).
+For details, see the trace-id section under [traceparent Header Field Values](#traceparent-header-field-values).
```

Comment:
This is unrelated the the PR, but this link was broken (md anchors are always lower case only)